### ISSUE TITLE
asset_statements delegation permisson

### ISSUE THREAD

**Issue by kimuradev** [GoogleChrome/android-browser-helper#59]
Looking at README, in order to enable multiple domains, we need to add the "asset_statements" as declared there: 

```
  <string name="asset_statements">
    [{
        \"relation\": [\"delegate_permission/common.share_location\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://github.com\"
        }
    }],
    [{
        \"relation\": [\"delegate_permission/common.share_location\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://www.google.com\"
        }
    }],
    [{
        \"relation\": [\"delegate_permission/common.share_location\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://www.wikipedia.com\"
        }
    }],
    </string>
```

But looking at source code, the delagate_permission is differente than is wrote in README. 

https://github.com/GoogleChrome/android-browser-helper/blob/master/demos/twa-multi-domain/src/main/res/values/strings.xml

```
  <string name="asset_statements">
    [{
        \"relation\": [\"delegate_permission/common.handle_all_urls\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://github.com\"
        }
    }],
    [{
        \"relation\": [\"delegate_permission/common.handle_all_urls\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://www.google.com\"
        }
    }],
    [{
        \"relation\": [\"delegate_permission/common.handle_all_urls\"],
        \"target\": {
            \"namespace\": \"web\",
            \"site\": \"https://www.wikipedia.com\"
        }
    }],
    </string>
```

The first one is using delegate_permission/**common.share_location** and the second one is using delegate_permission/**common.handle_all_urls**.

Questions:

1) Which one is the correct?
2) Does it matter to enable multi domains?

**Comment by andreban** [comment]
The correct one is `common.handle_all_urls`.

We have recently fixed this on #57. The README has been updated as well. Can you try reloading and see if you are still seeing the wrong one?

**Comment by kimuradev** [comment]
Right now both are with `common.handle_all_urls`. Thanks!
